### DISCUSSION TITLE
Azure kubelogin

### DISCUSSION THREAD

**Discussion by ahmadhusaini** [actions/virtual-environments#5598] [Installed software]
Hello Team, 

I believe kubelogin is a must-have for all agents. I have to install it every time I deploy helm charts to the AKS cluster using deployment principal managed identity to authenticate. Helm deployment should take seconds. However, the deployment is taking around 2 mins because I have to download kubelogin. 

I'd appreciate it if you consider my request. Thank you for your awesome work.

**Comment by miketimofeev** [comment]
Hi @ahmadhusaini!
Could you please explain why installing kubelogin in runtime takes so long? As I see [on the releases page](https://github.com/Azure/kubelogin/releases/) the pre-binaries are pretty small and downloading them shouldn't take much time

**Comment by sergeidavydov** [comment]
I'd like to add my vote to the request. It will be helpful to have kubelogin preinstalled.
